build.gradle.kts

```diff
@@ -19,6 +19,7 @@ import org.gradle.api.tasks.testing.logging.TestExceptionFormat
 plugins {
   java
   id("nebula.release") version "15.1.0"
+  id("io.codearte.nexus-staging") version "0.22.0"
```

Comment:
Why do we need this plugin?

Reply:
It can allow automating the repo close - for the first time publishing to maven central I'll run the command manually and if it works add it to the workflow.

Reply:
Neat!